components/hab/src/command/pkg/binlink.rs

```diff
@@ -53,21 +54,35 @@ pub fn start(
         )?;
         fs::create_dir_all(&dst_path)?
     }
+    let ui_symlinked =
+        format!(
+        "Symlinked {} from {} to {}",
+        &binary,
+        &pkg_install.ident(),
+        &dst.display(),
+    );
     match fs::read_link(&dst) {
         Ok(path) => {
-            if path != src {
+            if force && path != src {
                 fs::remove_file(&dst)?;
                 filesystem::symlink(&src, &dst)?;
+                ui.end(&ui_symlinked)?;
+            } else if path != src {
+                ui.warn(
+                    format!("Skipping symlink because {} already exists at {}. Use -f to overwrite",
```

Comment:
It would be a bit clearer to tell the user to use `--force` (i.e. the long form) in this message.

Additionally, with the `has pkg install` command, this would be reported and there currently isn’t such a flag which would be confusing. I guess we’d default to non-forcing on package install but tell the user to explicitly call `hab pkg binlink` in that case?